pulpcore/pulpcore/app/serializers/task.py

```diff
@@ -115,10 +115,17 @@ class WorkerSerializer(ModelSerializer):
     )
 
     online = serializers.BooleanField(
-        help_text='Whether the worker is online or not. Defaults to True.',
+        help_text=_('Whether the worker is online'),
+        read_only=True
+    )
+
+    gracefully_stopped = serializers.BooleanField(
+        help_text=_('If this worker is currently offline, this represents whether the worker \
```

Comment:
Similar here w/ the True and False values themselves called out in the docstring help.